mkt/developers/models.py

```diff
@@ -494,7 +494,7 @@ def to_string(self, type_=None):
 
         try:
             kw = dict(addon=addon, review=review, version=version, group=group,
-                      collection=collection, tag=tag, user=self.user.username)
+                      collection=collection, tag=tag, user=self.user.display_name)
```

Comment:
Yes, my point was, is there an test checking that there is no possibility of XSS through the user display name in the log viewer.